src/university/universityParts/Student.java

```diff
@@ -0,0 +1,136 @@
+package university.universityParts;
+
+import university.UniversityService;
+import university.exceptions.GradeOutOfRangeException;
+import university.exceptions.itemNotFoundExceptions.CourseNotFoundException;
+import university.exceptions.nullListExceptions.NullCourseListException;
+
+import java.text.DecimalFormat;
+import java.util.Random;
+
+/**
+ * The Student class represents the students of the university.
+ * Contains lists of courseGrades and relevant information about
+ * the student.
+ *
+ * @author Karine Gevorgyan
+ */
+
+public class Student {
+
+    final static DecimalFormat numberFormat = new DecimalFormat("#.0");
+    final static int MIN_GRADE = 0;
+    final static int MAX_GRADE = 10;
+    private String fullName;
+    private Course[] courses;
+    private String address;
+    private String phone;
+    private String email;
+
+    private Student(StudentBuilder student) {
+        this.fullName = student.fullName;
```

Comment:
builder pattern used incorrectly, the idea is to fill all Student's fields from builder